keras_nlp/models/t5/t5_multi_head_attention.py

```diff
@@ -12,18 +12,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import tensorflow as tf
-from tensorflow.compiler.tf2xla.python.xla import dynamic_slice
+import numpy as np
 
 from keras_nlp.backend import keras
+from keras_nlp.backend import ops
 
 
 def shape_list(tensor):
```

Comment:
we can probably just remove this whole function and use `ops.shape` where ever it was used.

Reply:
Fixed, sorry left over while debugging!